cli/src/clients/datafusion_helpers.rs

```diff
@@ -957,3 +958,43 @@ pub async fn get_invocation_journal(
     journal.reverse();
     Ok(journal)
 }
+
+#[derive(Debug, Clone, PartialEq, ArrowField, ArrowDeserialize)]
+pub struct StateKeysQueryResult {
+    service_name: Option<String>,
+    service_key: Option<String>,
+    key: Option<String>,
+    value: Option<Vec<u8>>,
```

Comment:
Why do we need to get the value?

Reply:
for generating the "version number" (that is the hash of key and value for all the state of a given service)

Reply:
It'd be nice if this can be done on the server side, value can be too big to fetch for such a large number of keys.